pkg/deploy/assets/env-development.json

```diff
@@ -139,7 +154,7 @@
             "apiVersion": "2020-08-01",
```

Comment:
We may need to remove the condition  `"[equals(parameters('ciCapacity'), 0)]"`

This block along with [Line 62](https://github.com/Azure/ARO-RP/pull/2238/files#diff-efb7c35dcb2ed4f45e17747bb9d1e87f5e1a99062a63d54c7100a16f16969adcR62) blocks the creation of vpn and public ip address. 

Moving forward with Simply Secure, even in CI deployments we need a VPN.

Reply:
That makes sense to me - generator and templates have been updated to remove dependency on ci capabilites for VPN